packages/node/package.json

```diff
@@ -16,6 +16,7 @@
     "@types/node": "^10.9.3",
     "dotenv-extended": "^2.3.0",
     "firebase-server": "^1.0.1",
+    "ganache-core": "2.3.3",
```

Comment:
what is this for

Reply:
Running an in-process ganache instead of externally setting up the ganache infra